doc/doxygen/src/getting-started.md

```diff
@@ -89,54 +175,6 @@ hex file in the `bin` folder of your application directory.
 Learn more about the build system in the
 [Wiki](https://github.com/RIOT-OS/RIOT/wiki/The-Make-Build-System)
 
-Building and executing an example           {#building-and-executing-an-example}
----------------------------------
-RIOT provides a number of examples in the `examples/` directory. Every example
-has a README that documents its usage and its purpose. You can build them by
-typing
-
-~~~~~~~~ {.sh}
-make BOARD=samr21-xpro
-~~~~~~~~
-
-or
-
-~~~~~~~~ {.sh}
-make all BOARD=samr21-xpro
-~~~~~~~~
-
-into your shell.
-
-To flash the application to a board just type
-
-~~~~~~~~ {.sh}
-make flash BOARD=samr21-xpro
-~~~~~~~~
-
-You can then access the board via the serial interface:
-
-~~~~~~~~ {.sh}
-make term BOARD=samr21-xpro
-~~~~~~~~
-
-If you are using multiple boards you can use the `PORT` macro to specify the
-serial interface:
-
-~~~~~~~~ {.sh}
-make term BOARD=samr21-xpro PORT=/dev/ttyACM1
-~~~~~~~~
-
-Note that the `PORT` macro has a slightly different semantic in `native`. Here
-it is used to provide the name of the TAP interface you want to use for the
-virtualized networking capabilities of RIOT.
-
-We use `pyterm` as the default terminal application. It is shipped with RIOT in
-the `dist/tools/pyterm/` directory. If you choose to use another terminal
-program you can set `TERMPROG` (and if need be the `TERMFLAGS`) macros:
-
-~~~~~~~~ {.sh}
-make -C examples/gnrc_networking/ term \
-    BOARD=samr21-xpro \
-    TERMPROG=gtkterm \
-    TERMFLAGS="-s 115200 -p /dev/ttyACM0 -e"
-~~~~~~~~
+[native]:            https://github.com/RIOT-OS/RIOT/wiki/Family:-native
```

Comment:
Rather link to `@ref boards_native`? I'm not sure if that page is (or will be kept) up to date. Maybe @jia200x knows?

Reply:
The scope of the migration was only for board entries. So in theory, this page _should_ be kept up to date.
We can open the discussion about migrating these family entries to Doxygen though

Reply:
Thank you! So I see the discussion here resolved. The question of migration is independent of this PR ;-)